docs/report/report.tex

```diff
@@ -1226,8 +1237,8 @@ \subsection{Meilenstein 6}
 \wrapfiguresafe{r}{0mm}{
   \includegraphics[width=0.45\textwidth]{noTagsFound.png}}{no tags found div}
 Auf der Startseite wurde dem User nun ein Hinweis angezeigt, falls dieser noch keine NFC-Tags angelegt hat. Hierfür wird beim Laden der NFC-Tags die Anzahl geprüft und im Falle von keinem vorhandenen Tags ein \textit{div} mit einem \textit{GIF} und eines Hinweistextes über \textit{v-if} angezeigt oder ausgeblendet.
-\wrapfiguresafe{r}{0mm}{
-  \includegraphics[width=0.45\textwidth]{HomescreenNoTags.jpeg}}{Startseite ohne Tags}
+
+\wrapfiguresafe{r}{0mm}{\includegraphics[width=0.45\textwidth]{HomescreenNoTags.jpeg}}{Startseite ohne Tags}
```

Comment:
das klappt so nicht. Das bild ist im Report jetztz auf Höhe des Topics "Button Refactor". Hat aber überhaupt nichts damit zu tun.

Reply:
Da war es vorher auch schon. Macht auch Sinn weil der Teil vorher das bespricht, aber 'zu kurz' für das Bild ist.
Diese Änderung setzt das nur in eine Zeile.

Reply:
Da stimme ich @julianhahn zu. Du kannst von mir aus die Abbildung von dem Codeschnipsel (Line 1237 & 1238) herausnehmen und stattdessen da dieses Bild reinsetzten :)